stdlib/2/__builtin__.pyi

```diff
@@ -28,7 +28,7 @@ class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
     __dict__ = ...  # type: Dict[str, Any]
-    __slots__ = ...  # type: Optional[Union[str, unicode, Iterable[Union[str, unicode]]]]
+    __slots__ = ...  # type: Union[str, unicode, Iterable[Union[str, unicode]]]
```

Comment:
I don't think so.
```
$ python2.7
Python 2.7.10 (default, Oct  6 2017, 22:29:07) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> class A(object):
...     __slots__ = None
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Error when calling the metaclass bases
    'NoneType' object is not iterable
```
(That's a new-style class. Old-style classes do allow `__slots__ = None`, but that's because they allow anything to be set in `__slots__` and ignore the value.)

Reply:
Ah, you are right. I didn't know old style classes don't use slots.